main.py

```diff
@@ -508,7 +508,7 @@ def main() -> None:
     parser.add_argument(
         "-j",
         "--libjs-test262-runner",
-        default="./Build/libjs-test262-runner",
+        default="./Build/_deps/lagom-build/test262-runner",
```

Comment:
I feel like we probably shouldn't provide a default here anymore, and perhaps even remove all cmake deps and let the outside deal with building the runner

Reply:
Yeah seems fine as well but that would require more changes for generating the test262 CI results

Reply:
Would it? We already build test-js and test-wasm (https://github.com/SerenityOS/serenity/blob/master/.github/workflows/libjs-test262.yml#L104), surely we could also just add test262-runner to the list and it would just work?

Reply:
Oh actually we need to add it there I guess, that would require merging at the same time.

But that still uses the CMake file in this repo currently as far as I understand.